internal/controller/postgrescluster/cluster_test.go

```diff
@@ -848,12 +850,30 @@ func TestPatroniLogSize(t *testing.T) {
 				StorageLimit: &tooSmall,
 			}}
 
-		size := reconciler.patroniLogSize(&cluster)
+		size := reconciler.patroniLogSize(ctx, &cluster)
 
 		assert.Equal(t, size, int64(25000000))
 		assert.Equal(t, len(recorder.Events), 1)
 		assert.Equal(t, recorder.Events[0].Regarding.Name, cluster.Name)
 		assert.Equal(t, recorder.Events[0].Reason, "PatroniLogStorageLimitTooSmall")
 		assert.Equal(t, recorder.Events[0].Note, "Configured Patroni log storage limit is too small. File size will default to 25M.")
 	})
+
+	t.Run("SizeUnsetOtelLogsEnabled", func(t *testing.T) {
```

Comment:
I'd like a test where both are set, so we can see the set Storage Limit win out

Reply:
Good call.